nifi-extension-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/services/azure/eventhub/AzureEventHubRecordSink.java

```diff
@@ -143,7 +143,7 @@ public class AzureEventHubRecordSink extends AbstractControllerService implement
 
     @Override
     protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
-        return PROPERTY_DESCRIPTORS;
+        return PROPERTIES;
```

Comment:
As I look through here it is clearly inconsistent whether we use the term properties, descriptors, or property descriptors.  However, we should be careful.  These things we're messing with in all these files are PropertyDescriptor objects.  They're not actually properties.  If we're going to establish more consistency we should consider using the term DESCRIPTOR or PROPERTY_DESCRIPTOR but probably not simply PROPERTIES

Reply:
How about PROPERTY_DESCRIPTORS (plural)?

Reply:
Yeah sorry I meant the plural form (what it was originally).